_data-prepper/common-use-cases/anomaly-detection.md

```diff
@@ -0,0 +1,210 @@
+---
+layout: default
+title: Anomaly detection with Data Prepper
+parent: Common use cases
+nav_order: 30
+---
+
+# Anomaly detection with Data Prepper
+
+You can use Data Prepper to train models and generate anomalies in near real-time on time-series aggregated events. You can generate anomalies either on events generated within the pipeline, or on events coming directly into the pipeline, like OpenTelemetry metrics. You can feed these tumbling window aggregated time-series events to the [Anomaly detector]({{site.url}}{{site.baseurl}}/data-prepper/pipelines/configuration/processors/anomaly-detector/) processor, which trains a model and generates anomalies with a grade score. Then write the anomalies to a separate index to create document monitors and trigger fast alerting.
+
+## Metrics from logs 
+
+The following pipeline receives logs using an HTTP source like FluentBit, extracts important values from the logs by matching the value in the `log` key against the grok common Apache log pattern, and then forwards the grokked logs to both the `log-to-metrics-pipeline sub-pipeline` and to an OpenSearch index named `logs`.
```

Comment:
What you have as "common Apache log pattern" is called "Apache Common Log Format".

We may want to link it to this page: https://httpd.apache.org/docs/2.4/logs.html#accesslog